strawberryfields/backends/tfbackend/states.py

```diff
@@ -55,124 +44,72 @@ def __init__(
         self._str = "<FockStateTF: num_modes={}, cutoff={}, pure={}, batched={}, hbar={}>".format(
             self.num_modes, self.cutoff_dim, self._pure, self._batched, self._hbar
         )
-        self._graph = graph
-        self._eval = eval
-
-    def _run(self, t, **kwargs):
-        """
-        Numerically evaluates Tensor `t` based on the arguments in **kwargs.
-
-        Args:
-            t: Tensor (or iterable of Tensors) to (potentially) numerically evaluate.
-            **kwargs: can be used to pass a session or a feed_dict. Otherwise a temporary session
-                and no feed_dict will be used.
-
-        Returns:
-            The numerical evaluation of the Tensor t. Type is the same as the dtype of t
-        """
-        with self._graph.as_default():
-            if "eval" not in kwargs:
-                kwargs[
-                    "eval"
-                ] = (
-                    self._eval
-                )  # when the caller did not specify, use the default behaviour of this instance
-            evaluate, session, feed_dict, close_session = _check_for_eval(kwargs)
-
-            if evaluate:
-                t = session.run(t, feed_dict=feed_dict)
-                if close_session:
-                    session.close()
-            return t
 
     def trace(self, **kwargs):
         r"""
         Computes the trace of the state. May be numerical or symbolic.
 
         Args:
```

Comment:
Same as my other comment above; should the `Args:` still be there even though it's empty?

Reply:
Removed!